src/utils.js

```diff
@@ -9,11 +9,11 @@ export const apiRequest = (url, options = {}) => {
   return fetch(url, options)
     .then(res => {
       if (res.status >= 200 && res.status < 300) {
-        if (jsonContentTypes.some(contentType => res.headers.get('Content-Type').indexOf(contentType) > -1)) {
+        if (res.status === 204) {
+          return res;
+        } else if (jsonContentTypes.some(contentType => res.headers.get('Content-Type').indexOf(contentType) > -1)) {
           return res.json();
         }
-
-        return res;
```

Comment:
It doesn't seem like this return was ever reachable since this was previously expecting every response to have `Content-Type` (in the success family)